test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/CustomUriLiteralPrefixesTests.cs

```diff
@@ -327,10 +327,30 @@ public void CustomUriLiteralPrefix_CannotParseWithCustomLiteralPrefix_IfBuiltInP
         [Fact]
         public void CustomUriLiteralPrefix_CannotParseTypeWithWrongLiteralPrefix()
         {
+            // Ensure the prefix under test is registered. Handle the exception gracefully if

```

Comment:
Is there a way to unregister the instance during test clean-up?

Reply:
This are no ways to unregister the static instance. At this point we are not sure where the interference is coming from, but ensure proper setup for test here is the right thing to do. I have check other places, and they are all using unique prefixes, therefore should be no other issues.

---
In reply to: [141485235](https://github.com/OData/odata.net/pull/962#discussion_r141485235) [](ancestors = 141485235)